pyproject.toml

```diff
@@ -108,6 +108,11 @@ ignore-init-module-imports = true
     "S603",  # todo: `subprocess` call: check for execution of untrusted input
     "S605",  # todo: Starting a process with a shell: seems safe, but may be changed in the future; consider rewriting without `shell`
     "S607",  # todo: Starting a process with a partial executable path
+    "PT004",  # todo: Fixture `tmpdir_unittest_fixture` does not return anything, add leading underscore
+    "PT011",  # todo: `pytest.raises(ValueError)` is too broad, set the `match` parameter or use a more specific exception
+    "PT012",  # todo: `pytest.raises()` block should contain a single simple statement
+    "PT015",  # todo: Assertion always fails, replace with `pytest.fail()`
+    "PT019",  # todo: Fixture `_` without value is injected as parameter, use `@pytest.mark.usefixtures` instead
```

Comment:
will address remaining in separate PRs, or we can ask contributors to help here...
cc: @aniketmaurya @JustinGoheen

Reply:
@Borda can we create a single issue users can comment on to volunteer for a particular fix? similar to how the type hint issues were resolved last year.

Reply:
@Borda I can create the issue. Can you mark each of the tasks in the issue with a difficulty level after I create it?